include/wlr/render/gles2.h

```diff
@@ -30,8 +30,17 @@ struct wlr_gles2_texture_attribs {
 	bool has_alpha;
 };
 
+struct wlr_buffer;
+struct wlr_gles2_buffer_attribs {
+	GLuint fbo;
```

Comment:
Since the attribs struct only has a single field, maybe we can just have a function that directly returns the FBO?

Reply:
This is to prevent the need to add new content in the future, I don't know if there will always be only fbo in the future

Reply:
Done